test/unit/remotestorage-suite.js

```diff
@@ -476,30 +328,6 @@ define(['require', 'tv4', './build/eventhandling', './build/util'],
         }
       },
 
-      {
-        desc: "#connect throws DiscoveryError on empty href",
-        run: function(env, test) {
-          env.rs.on('error', function(e) {
-            if (e instanceof RemoteStorage.DiscoveryError) {
-              test.done();
-            }
-          });
-          env.rs.connect('someone@somewhere');
-        }
```

Comment:
This one wasn't really testing anything at all, because the implementation there is specifically to the test file/source itself. But we cover the case in https://github.com/remotestorage/remotestorage.js/blob/fceb3f47def6e6f333dc3210ac6af15f33a396d3/test/unit/discover.test.ts#L125 also.